pkg/descheduler/strategies/topologyspreadconstraint.go

```diff
@@ -183,6 +190,16 @@ func RemovePodsViolatingTopologySpreadConstraint(
 	}
 }
 
+// checkIdenticalConstraints checks if already had an identical TopologySpreadConstraint in namespaceTopologySpreadConstraints map
+func checkIdenticalConstraints(newConstraint v1.TopologySpreadConstraint, namespaceTopologySpreadConstraints map[v1.TopologySpreadConstraint]struct{}) bool {
```

Comment:
`s/checkIdenticalConstraints/hasConstraint` so it is clear from the function name what is getting checked, also given `namespaceTopologySpreadConstraints` is expected to be seen as a set prefixing the function name with `has` makes it more "set-ish".